### DISCUSSION TITLE
[Solved] GitHub Copilot Chat returns 404

### DISCUSSION THREAD

**Discussion by Haoxiang-Hou** [community/community#110196] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

I can not use github copilot chat, while my github copilot works well.
![image](https://github.com/community/community/assets/127498004/24f5c13e-0264-4809-be4e-94a54db5c529)
When I use github copilot to code, it is ready and works very well.
![image](https://github.com/community/community/assets/127498004/8068ea48-e404-4aa5-80f7-b466a9156a63)
However, when I try to chat with github copilot chat, it will say "unhandled status from server: 404 404 page not found". That's  annoying.
My IDE is VS Code and all of the extensions are latest. Ping github.com is OK in my server and I have logined in.

**Comment by allisonee** [comment]
+1 Same here, tried singing out and signing back in as seen on some blogs and it didn't work

**Comment by benjamin-lykins** [comment]
Nope 😮‍💨

**Comment by davecheney** [comment, marked as answer]
@Haoxiang-Hou @allisonee @benjamin-lykins you are running an outdated chat client. Please follow these steps.

1. Uninstall VSCode-insiders, the -insiders build is no longer required for Copilot Chat.
2. Install the latest version of VSCode, older versions of VSCode do not support features required for the most recent versions of the Copilot and Copilot Chat extensions.
3. Install the Copilot extension from the marketplace, this will install the Copilot Chat extension automatically. Once installed, if the extension requires an update or restart, please do so until VSCode is happy.

**Reply by benjamin-lykins** [reply]
Ah! That makes sense. Looks like my vscode hadn't updated since I originally installed it. Had to uninstall and reinstall it.

**Reply by niceday711** [reply]
Thank u, bro

**Reply by Estrellajer** [reply]
thank you, i never thought to update my vscode

**Reply by Bing72** [reply]
thx!!!

**Comment by mcmoe** [comment]
Hey @davecheney, I had the `command 'github.copilot.interactiveSession.feedback' not found` faced in https://github.com/orgs/community/discussions/77195 on a new installation. I installed the extension from the marketplace and it is up to date. I was already signed in to my GitHub account prior to the extension installation. After I signed out and signed back in, I stopped getting the error and chat started to work. I thought I'd drop this here in case someone faces something similar - as my issue was not due to an outdated version of the extension.

**Comment by Lixc23tsinghua** [comment]
Thank you!

**Comment by gf-mse** [comment]
Same here -- `unhandled status from server: 404 404 page not found`, VSCode update / Copilot reinstall did not help.

* VSCode version:

```
Version: 1.79.2 (Universal)
Commit: 695af097c7bd098fbf017ce3ac85e09bbc5dda06
Date: 2023-06-14T08:58:52.392Z (9 mos ago)
Electron: 22.5.7
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Darwin arm64 23.4.0
```

Copilot extension version:

```
Published
6/30/2021, 00:26:17
Last released
3/21/2024, 12:38:56
Last updated
3/21/2024, 13:23:23
Identifier
github.copilot
```

**Reply by davecheney** [reply]
What is the version of your chat extension please?

**Comment by gf-mse** [comment]
This seems to be the most recent one available:
```
Published
4/27/2023, 17:13:54
Last released
3/21/2024, 09:09:45
Last updated
3/21/2024, 13:47:24
Identifier
github.copilot-chat
```

( I was switching between release and pre-release versions without much luck )

**Reply by gf-mse** [reply]
To add, I just tested Zed with Copilot autocomplete feature -- seems to work Ok, so I'm inclined to suspect that this is broken on VSCode side

**Reply by adamseoul** [reply]
Is this resolved for you now? I get the same error

**Reply by gf-mse** [reply]
Nope, but I have a colleague using a Windows OS, and he does not seem to have the same issue. So it could be somehow a MacOS version problem.

**Reply by eg271** [reply]
Same issue here on macos with VSCode:

```
Version: 1.80.0 (Universal)
Commit: 660393deaaa6d1996740ff4880f1bad43768c814
Date: 2023-07-04T13:39:33.766Z (9 mos ago)
Electron: 22.3.14
ElectronBuildId: 21893604
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Darwin arm64 23.4.0
```

Github Chat
```
Published
4/27/2023, 02:13:54
Last released
4/19/2024, 05:11:14
Last updated
4/19/2024, 13:09:29
Identifier
github.copilot-chat
```

**Reply by eg271** [reply]
So, it looks like MacOS was blocking updates to VSCode.  Had to delete VScode, redownload it and move it to the Applications folder. Now Github chat is working

**Reply by seanfuture** [reply]
Same issue - MacOS blocking updates.  Thanks @eg271.  After update, fixed ofc.
